.github/CONTRIBUTING.md

```diff
@@ -42,21 +42,32 @@ Gradle uses pull requests for contributions. Fork [gradle/gradle](https://github
     git config user.name 'First Last'
     git config user.email user@example.com
 
+### IntelliJ
+
 You can generate the IntelliJ project by running
 
-    ./gradlew idea
+    gradlew idea
+
+### Eclipse
 
-You can generate the Eclipse projects by running
+You can generate the Eclipse project <ith the following steps
```

Comment:
I think you mean "with" instead of "<ith". Please add a ":" at the end of this line.

Reply:
Sure. will fix it